app/src/main/java/com/example/leagueoflegends/view/SplashActivity.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.leagueoflegends.view
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import com.example.leagueoflegends.R
+
+class SplashActivity : AppCompatActivity() {
```

Comment:
this splash feature can be an activity if you have to download configurations before starting the app, but in this case, you can put a layout file into the manifest and it will do.
its okay, I know we haven't done splash screens before, I just wanted to tell